### DISCUSSION TITLE
does the standalone rely on anything from the project?

### DISCUSSION THREAD

**Discussion by MarlonMrN** [security-code-scan/security-code-scan#225] [Q&A]
Hi and first of all, thanks for this project!!

From the testing I made, the .zip version intended to scan older .netframework4.x works really really well!!

I'm migrating my .net core scans from the "package adding" version to the new standalone scanner. With the previous way, I usually faced issues with conflicts between scanner and project depdencies, like the the compiler version referenced in the project don't having everything the scanner needed and usually throwing some warnings and not working properly.

With this new standalone version, is it completely separated from the project dependencies? Can I scan older .net core applications, the ones that reference netcoreapp2.1/3.0/3.1 in the target framework... ? (assuming I have .net 5.0 installed in the environment) 

Thanks!!

**Comment by JarLob** [comment]
My understanding was that, even there are some issues like https://github.com/dotnet/roslyn/issues/52177 (is not available in prod yet), adding the analyzer as nuget is the most reliable way. Building with a standalone tool often fails to load some projects, although from my experience the 4.x zip version is more often successful than the nuget global tool counterpart.
